app/js/controllers/events.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('EventsCtrl', function($scope, $location, Restangular) {
+	$scope.loadingPromise = Restangular.one('events/next-10-publicly')
```

Comment:
The URI to request would be `/upcoming-publicly?page[limit]=10`

The idea is to have two query parameters eventually, that'll be supported everywhere: `page[limit]` and `page[offset]`, even though for now we're only supporting `page[limit]` and only on this one route.

Naming the query parameters with this `page[....]` format doesn't make them officially related from the URI spec's pov, but it's become a convention for showing that they are. JSON API encourages this convention by designating all `page[...]` parameters as parameters to be used for pagination, and express encourages it too by parsing all parameters with the same prefix into a common object. That is: `xyz.com?page[offset]=0&page[limit]=10&name=x` is put into `req.query` as:

```
{
  "name": "x",
  "page": { "offset": 0, "limit": 10 }
}
```

rather than:

```
{
  "name": "x",
  "page[offset]": 0, 
  "page[limit]": 10
}
```

As an aside, the `[` and `]` aren't technically allowed characters in the query part of a uri under the uri syntax RFC. But the entire world disregards that and/or clients automatically prevent encode those characters (which is allowed) and everything works fine.